packages/kbn-search-index-documents/components/documents_list.test.tsx

```diff
@@ -1,27 +1,24 @@
 /*
  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
  * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
  */
 
-import { setMockValues, setMockActions } from '../../../../../__mocks__/kea_logic';
-
 import React from 'react';
 
 import { shallow } from 'enzyme';
```

Comment:
Do we want to use `react-testing-library` instead of enzyme? I guessed we don't considering that it's a package not dedicated or serverless plugin

Reply:
If we can then we do, yes. Whether it's a package or plugin doesn't really matter.

Reply:
Added in 2902ee46b261cf87c3c6ab5109b635978493f950